pyproject.toml

```diff
@@ -19,7 +19,7 @@ keywords = ['exasol', 'udf', 'script-languages']
 
 [tool.poetry.dependencies]
 python = ">=3.10,<4"
-exasol-script-languages-container-tool = "^1.0.0"
+exasol-script-languages-container-tool = "^1.1.0"
```

Comment:
We probably also should increase the SLC version

Reply:
You mean the major version?